.github/workflows/pr-created.yaml

```diff
@@ -16,5 +16,5 @@ jobs:
     uses: kubescape/workflows/.github/workflows/incluster-comp-pr-created.yaml@main
     with:
       CGO_ENABLED: 0
-      GO_VERSION: "1.19"
+      GO_VERSION: "1.20"
```

Comment:
Is it 20?
Please update the go mod as well

Reply:
@dwertent

Reply:
yes, it will trigger the new WF

Reply:
I removed this dependency... The tests will run when with the label

Reply:
With the release label